src/__tests__/CompareResults/CompareResultsTable.test.tsx

```diff
@@ -10,12 +11,32 @@ import {
 } from '../../reducers/FilterCompareResultsSlice';
 import { ActiveFilters, FilteredResults } from '../../types/types';
 import getTestData from '../utils/fixtures';
-import { render, store } from '../utils/setupTests';
+import { render, renderWithHook, store } from '../utils/setupTests';
 import { screen, waitFor } from '../utils/test-utils';
 
 const { testCompareData } = getTestData();
 
 describe('Compare Results Table', () => {
+  it('should filter results hook', async () => {
```

Comment:
this is not a valid test yet. Trying to understand how [renderHook ](https://testing-library.com/docs/react-testing-library/api/#renderhook) works in this case.